source/plot/SvgPlot.ooc

```diff
@@ -34,7 +34,10 @@ SvgPlot: class {
 		this datasets free()
 		this xAxis free()
 		this yAxis free()
-		this colorList free()
+		if (this colorList) {
+			this colorList _count = 0
```

Comment:
This seems.. weird. Why not set `freeContent` to `false` when creating the `colorList`?